### DISCUSSION TITLE
媒體：摩根士丹利開始新一輪裁員 涉及香港和大陸

### DISCUSSION THREAD

**Discussion by Floriaand** [1101-GS4719-C/Python-Community-GS4719-C#834] [General]
最近美國金融業又傳出裁員，沒有被裁員的也會減薪，再加上前陣子台積電鼓勵員工休假，可以感受到真的很不景氣，製造業跟金融業都是，今年因為一直升息又提高了存款準備率，讓銀行業資金不足，放款困難。我覺得現在通膨這麼嚴重，還是因為這兩年一直撒錢，讓市場上的錢太多了，導致物價一直漲，但升息這麼多真的很傷經濟。
網址：https://www.google.com/amp/s/m.hk.investing.com/news/stock-market-news/article-265047%3fampMode=1

**Comment by CPCpyt** [comment]
總結來說，在貨幣政策的影響下，經濟衰退是必然的趨勢。
如何調整資產配置面對未來的衰退是很重要的課題。
